app/Providers/AppServiceProvider.php

```diff
@@ -206,6 +206,182 @@ function.
+                                        'url' => '/visit/whos-visiting/adult-groups-2',
+                                    ],
+                                ],
+                            ],
+                            [
+                                'name' => 'Mobile App',
+                                'url' => '/visit/explore-on-your-own/mobile-app-audio-tours',
+                            ],
+                            [
+                                'name' => 'Ryan Learning Center',
+                                'url' => '/ryan-learning-center',
+                            ],
+                        ],
+                    ],
+                    [
+                        'name' => 'Exhibitions',
+                        'children' => [
+                            [
+                                'name' => 'Current',
+                                'url' => route('exhibitions'),
+                            ],
+                            [
+                                'name' => 'Upcoming',
+                                'url' => route('exhibitions.upcoming'),
+                            ],
+                            [
+                                'name' => 'Archive',
+                                'url' => route('exhibitions.history'),
+                            ],
+                        ],
+                    ],
+                    [
+                        'name' => 'Art &amp; Artists',
+                        'description' => 'Explore the works in our collection and delve deeper into their stories.',
+                        'image' => url('/images/navigation-thumbnail-art-and-artists.jpg'),
+                        'children' => [
+                            [
+                                'name' => 'Artworks',
+                                'url' => route('collection'),
+                            ],
+                            [
+                                'name' => 'Articles &amp; Videos',
+                                'url' => route('articles_publications'),
+                            ],
+                            [
+                                'name' => 'Research',
+                                'url' => route('collection.research_resources'),
+                                'children' => [
+                                    [
+                                        'name' => 'Library',
+                                        'url' => '/library',
+                                    ],
+                                    [
+                                        'name' => 'Archival Collections',
+                                        'url' => '/archival-collections',
+                                    ],
+                                    [
+                                        'name' => 'Collection Information',
+                                        'url' => '/collection-information',
```

Comment:
Figma has this as "Research Information"

Reply:
Confirming with Josh that what the Figma lists as "Research Initiatives" is what the copy document has listed as "Collection Information".

Reply:
From Josh:
> The figma is the design reference. This spreadsheet represents the actual content